diffusers/training_example.ipynb

```diff
@@ -0,0 +1,3955 @@
+{
```

Comment:
<p>Change "Pipeline" to <code>DiffusionPipeline</code>, also this is really verbose "object which you can pass your model and noise schedule into and you can then generate from the model with the appropriate sampling procedure" try trimming and clarifying.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>We can try:</p><p> </p><p>"Diffusers has a <code>DiffusionPipeline</code> class, where you can pass in a model and scheduler to directly generate samples in a few lines of code. "</p>

Reply:
Thanks for the suggestion, updated...